### DISCUSSION TITLE
BOARD707 смотреть здесь

### DISCUSSION THREAD

**Discussion by songzhishuo** [board707/w80x_arduino#14] [General]
Привет, я очень рад видеть, что вы собираетесь получить и поддерживать проект W806 в сообществе.Я также внес некоторый код в этот проект раньше.Я думаю, вы наверняка видели беспорядочные операции, написанные в таких интерфейсах, как digitalWrite, который я написал.Я ограничен и позволяю вам видеть мусорный код, который я пишу.Я хочу продолжить этот проект вместе с вами позже. @board707

**Comment by board707** [comment]
Здравствуйте, буду рад поработать с Вами. 
Если Вам удобнее - Вы можете писать по-английски.

**Reply by songzhishuo** [reply]
oh this is really great，How to get in touch?  Telegram ？ wechat ？email ？

**Comment by AnatolSher** [comment]
You are welcome! We really need a resident of China :)

**Reply by songzhishuo** [reply]
oh this is really great，How to get in touch? I've been using the arduino IDE，but w80x is the first library I contributed to writing。So you see that the submission code in the previous warehouse is confusing, I am very sorry

**Reply by AnatolSher** [reply]
![image](https://github.com/board707/w80x_arduino/assets/121094544/848827c8-eba0-4c26-96af-cdc7c5fe69cd)

**Reply by board707** [reply]
Анатолий что это? неужели тут реально что-то закодировано? :)

**Reply by AnatolSher** [reply]
мое мыло. Можно здесь сделать https://ru.qrcodechimp.com/qr-code-generator-for-text

**Reply by songzhishuo** [reply]
I saw this picture and thought it was wechat's qr

**Reply by AnatolSher** [reply]
WeChat does not work in Russia. Unfortunately, I was unable to register there. :)

**Reply by songzhishuo** [reply]
why？

**Reply by AnatolSher** [reply]
Because only Chinese citizens can use WeChat, CSDN, Baidu e.t.c. Our phone numbers are not valid in China zone

**Reply by songzhishuo** [reply]
I am really ignorant, I thought wechat can still be used outside of China. So can you use the Telegram APP?

**Reply by board707** [reply]
>  So can you use the Telegram APP?

Sorry, but I not using Telegram.
It's probably best to write directly to the discussion.

**Comment by songzhishuo** [comment]
So do you need my help to write something now? I still have the w806 and w801 boards that I used to help nulllab develop.

**Reply by AnatolSher** [reply]
YES! :)
Can you write a FastSPI class based on SDIO?
See http://ask.winnermicro.com/article/76.html for detail
FastSPI should be  compatible with the Arduino API as possible.

**Reply by board707** [reply]
> So do you need my help to write something now? 

At the [end of the Readme ](https://github.com/board707/w80x_arduino#w80x_duino-development)file you can see a list of what is done and what is not.
I would ask you to test what we have already done - there will surely be a lot of errors.

Or perhaps you can offer something of your own. For example, I am very interested in examples of using DMA

**Reply by songzhishuo** [reply]
embarrassed， I dont know whats mean is FASTSPI。I browsed the repository and saw that SPI's library had completed commits。

**Reply by songzhishuo** [reply]
I can write a library for Servo

**Reply by AnatolSher** [reply]
OK. 

P.S. A little bit later I will write what I mean by FastSPI. Further discussion required

**Reply by board707** [reply]
>I can write a library for Servo

It will be good.
Please write here about all found errors or open an issue.
